src/computation/logic/distribute-levelling-seats.ts

```diff
@@ -21,75 +100,124 @@ export function distributeLevelingSeats(
             votes: partyResults[partyCode].votes,
             percentage: -1,
         };
-    });
-
-    // Filter out parties with less votes than the threshold or who did not gain any seats from the national distribution
-    levelingParties = levelingParties.filter(
-        (party) => partyResults[party.partyCode].percentVotes >= payload.electionThreshold
-    );
-
-    let seatsToDistribute = payload.levelingSeats;
-    levelingParties.forEach((party) => {
-        seatsToDistribute += partyResults[party.partyCode].districtSeats;
-    });
+        levelingParties.push(party);
+    }
 
     // Compute the distribution of the total number of seats on the whole country
     const nationalDistribution = distributeSeats(
         payload.algorithm,
         payload.firstDivisor,
         Number.MIN_SAFE_INTEGER,
         seatsToDistribute,
+        totalVotes,
         levelingParties
     );
 
-    // Filter out parties who did not gain any seats from the national distribution
-    levelingParties = levelingParties.filter(
-        (party) => nationalDistribution.seatsWon[party.partyCode] >= partyResults[party.partyCode].districtSeats
+    // Filter out parties that did not gain any seats in the new distribution
+    const filteredLevelingPartyCodes = levelingPartyCodes.filter(
+        (p) => nationalDistribution.seatsWon[p] > partyResults[p].districtSeats
```

Comment:
The logic here changed, is it intentional? `>=` vs `>`

Reply:
Yes, only parties that **gain** seats in the national distribution should be in the draw for levelling seats.